listenbrainz/webserver/views/api_tools.py

```diff
@@ -49,6 +52,8 @@ def insert_payload(payload, user, listen_type=LISTEN_TYPE_IMPORT):
         _send_listens_to_queue(listen_type, augmented_listens)
     except (APIInternalServerError, APIServiceUnavailable) as e:
```

Comment:
picky: why do we catch this and then immediately raise it again without making a change? This is no different to removing these two lines

Reply:
Agreed. will remove.